doc/rtd/index.rst

```diff
@@ -87,8 +87,9 @@ projects, contributions, suggestions, fixes and constructive feedback.
 
    tutorial/index
    howto/index
-   explanation/index
    reference/index
+   explanation/index
+
```

Comment:
This seems a bit unintuitive to me, but maybe that is just because I'm used to reference last. Was this intentional?